doc/modules/model_evaluation.rst

```diff
@@ -62,6 +62,7 @@ Scoring                           Function
 'balanced_accuracy'               :func:`metrics.balanced_accuracy_score`           for binary targets
 'average_precision'               :func:`metrics.average_precision_score`
 'brier_score_loss'                :func:`metrics.brier_score_loss`
+'calibration_loss'                :func:`metrics.calibration_loss`
```

Comment:
to respect the convention higher is better we should maybe call it

`neg_calibration_error`

thoughts anyone?